websites/P/PresenceDB/presence.ts

```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({
+    clientId: "840759396103749633"
+}),
+{ language } = window.navigator;
+
+function getTranslation(name: string) {
+    switch(name) {
+        case "home":
+            switch(language) {
+                case "de":
+                    return "Durchsucht die Startseite";
+                default:
+                    return "Browsing home page";
+            }
+        case "search":
+            switch(language) {
+                case "de":
+                    return "Sucht nach etwas";
+                default:
+                    return "Searching for something";
+            }
+        case "faq":
+            switch(language) {
+                case "de":
+                    return "Durchsucht das FAQ";
+                default:
+                    return "Browsing FAQ page";
+            }
+        case "activity":
+            switch(language) {
+                case "de":
+                    return "Schaut sich Aktivität an";
+                default:
+                    return "Viewing activity";
+            }
+        case "user":
+            switch(language) {
+                case "de":
+                    return "Schaut sich Benutzer an";
+                default:
+                    return "Viewing user";
+            }
+        case "settings":
+            switch(language) {
+                case "de":
+                    return "Durchsucht die Einstellungen";
+                default:
+                    return "Browsing settings page";
+            }
+        case "view":
+            switch(language) {
+                case "de":
+                    return "Seite anzeigen";
+                default:
+                    return "View page";
+            }
+    }
+}
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "logo",
+        buttons: [
+            {
+                label: getTranslation("view"),
+                url: window.location.href
```

Comment:
make sure this isnt there on the homepage, since buttons to the homepage are not allowed.